src/crashlytics/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { FirebaseError } from "../error";
+
+export enum PLATFORM_PATH {
+  ANDROID = "topAndroidDevices",
+  IOS = "topAppleDevices",
+}
+
+export function parseProjectNumber(appId: string): string {
+  const appIdParts = appId.split(":");
+  if (appIdParts.length > 1) {
+    return appIdParts[1];
+  }
+  throw new FirebaseError("Unable to get the projectId from the AppId.");
+}
+
+export function parsePlatform(appId: string): PLATFORM_PATH {
+  const appIdParts = appId.split(":");
+  if (appIdParts.length < 3) {
+    throw new FirebaseError("Unable to get the platform from the AppId.");
+  }
+
+  if (appIdParts[2] === "android") {
```

Comment:
Can we allow for case insensitiveness here?

Reply:
I'm going to leave this as is since we are extracting this from the app id and the app id format only allows lowercase